spec/features/tutorials/show_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+describe "tutorial show page" do
+  it "does not throw error if there are no videos attached to tutorial" do
+    tutorial = create(:tutorial)
+
+    visit "/tutorials/#{tutorial.id}"
+
+    expect(page).to have_content("There are no videos.")
```

Comment:
We'll just have to verify this test passes after visiting a tutorial with no videos.